tools/calibration_scripts_system/README.md

```diff
@@ -0,0 +1,67 @@
+# OOI Instrument Parsers
+
+ The following scripts extract calibration information from various instrument on the OOI Cabled Array.
+ They allow for the automated creation of calibration files in the format requested by ooi-integration.
+ Serial numbers and asset tracking numbers can be found in the [asset-management](
+ https://github.com/ooi-integration/asset-management) repository.
+
+These scripts are based on those written by Dan Mergens, who developed the original scripts which can still be found in the [ooi-tools](https://github.com/oceanobservatories/ooi-tools/tree/master/instrument/calibration) repository.
+
+## Getting Started
+
+Clone the repository into your local machine. In the repository, there are a set of directories of each of the supported instrument types.
+
+### Prerequisites
+
+* Python 2.7
+* Linux terminal emulator
+
+If you do not already have Python installed on your computer, you will need to install it to make this program work.
+On Linux, run the following command.
+```
+sudo apt-get install python2.7
+```
+On MacOS, if you have Homebrew, call this command
+```
+brew install python2.7
+```
+On Windows, install Cygwin or preferred terminal emulator. Then install Python.
+
+## Running the code
+
+Navigate to the repository and in whatever terminal. There are a series of folders which contain the parsers and folders. Put the calibration files in the "Manufacturer Cal Files".
+
+Run the script by calling the corresponding parser.
+
+```
+./ctd_cal_parser.py
```

Comment:
`python ctd_cal_parser.py` will execute the parser even if the execution bit is not set on the filename and will also work on Windoze.